tests/test_generate.py

```diff
@@ -36,20 +45,14 @@ def setUp(self):
             ('pb369-jats.xml', 'pb-pubmed-369-20170717071707.xml',
              'pb', self.default_pub_date))
 
-    def read_file_content(self, file_name):
```

Comment:
The same could have been achieved using:
```
@staticmethod
def read_file_content(file_name):
    ...
```
If you still wanted to using it as a class method, but either is fine. One benefit being if you were to inherit from `TestGenerate` elsewhere (I don't know if you do) you would have `self.read_file_content()` available without having to import it again. 

I normally handle all generic file parsing in my test fixtures and pass in the actual parsed values to the test cases, 
isolates the logic a bit more and reduces unecessery multiple parses of the same file if the contents doesn't change.

This is just opinion, theres nothing technically wrong with you current approach.

Reply:
I see, I didn't completely understand ``@staticmethod`` and its uses. Tests in other libraries I have found these fixture reading functions are helpful to live outside of the test classes and they can be imported anyway, I think one place I put these helper functions right in ``__init__.py`` in the tests folder. I can follow what ever people prefer if there becomes a preference.